jts/README-source.adoc

```diff
@@ -154,78 +143,51 @@ Make a copy of this {productName} directory structure to use for the second serv
-
-Start server 1:
-```
-export WILDFLY_HOME_1=...
-$WILDFLY_HOME_1/bin/standalone.sh -c standalone-full.xml -Djboss.tx.node.id=UNIQUE_NODE_ID_1 &
-# enable JTS mode
-cd <quickstarts repo>/jts
-$WILDFLY_HOME_1/bin/jboss-cli.sh --connect --file=configure-jts-transactions.cli
-```
-
-Stop server 1 and create second server by copying the installation directory:
-```
-cp -r $WILDFLY_HOME_1 ${WILDFLY_HOME_1}-2
-export WILDFLY_HOME_2=${WILDFLY_HOME_1}-2
-```
-
-Configure postgresql on server 1 only as a module:
-```
-mkdir -p $WILDFLY_HOME_1/modules/org/postgresql/main
-# download postgresql-42.6.0.jar or later and install it into the server:
-cp <downloads>/postgresql-42.6.0.jar $WILDFLY_HOME_1/modules/org/postgresql/main
-```
-Create a module.xml for postgresql with the following contents:
-```
-$ cat $WILDFLY_HOME_1/modules/org/postgresql/main/module.xml
-<?xml version="1.0" encoding="UTF-8"?>
-<module xmlns="urn:jboss:module:1.1" name="org.postgresql">
-    <resources>
-        <!--resource-root path="postgresql-42.1.4.jre7.jar"/-->
-        <resource-root path="postgresql-42.6.0.jar"/>
-    </resources>
-    <dependencies>
-        <module name="javax.api"/>
-        <module name="javax.transaction.api"/>
-    </dependencies>
-</module>
-```
-
-Add the datasource to $WILDFLY_HOME_1/standalone/configuration/standalone-full.xml
-When you have done this the final changes to standalone-full.xml should be:
-
-```
-$ diff $WILDFLY_HOME_1/standalone/configuration/standalone-full.xml $WILDFLY_HOME_2/standalone/configuration/standalone-full.xml
-137,142d136
-<                 <datasource jndi-name="java:jboss/datasources/PostgresDS" pool-name="PostgresDS" enabled="true" use-java-context="true" statistics-enabled="${wildfly.datasources.statistics-enabled:${wildfly.statistics-enabled:false}}">
-<                     <connection-url>jdbc:postgresql://localhost:5432/postgres</connection-url>
-<                     <driver>postgresql</driver>
-<                     <security user-name="sa" password="sa"/>
-<                 </datasource>
-<
-147,149d140
-<                     <driver name="postgresql" module="org.postgresql">
-<                         <xa-datasource-class>org.postgresql.xa.PGXADataSource</xa-datasource-class>
-<                     </driver>
-
-```
+The {productName} server 1 need to be configured to be able to connect to the database created previously in the xref:configure_the_postgresql_database_for_use_with_this_quickstart[Createa a PostgreSQL Database] section.
+First of all, a JDBC driver needs to be installed as https://docs.wildfly.org/30/Developer_Guide.html#Class_Loading_in_WildFly[jboss module].
+
+The following command downloads the PostgreSQL driver automatically through Maven.
+
+1. Open a new terminal, navigate to the root directory of this quickstart, and run the following command, replacing WILDFLY_HOME with the path to your server:
+[source,sh,subs="+quotes,attributes+",options="nowrap"]
++
+----
+cd $QUICKSTART_HOME;
```

Comment:
You already navigated according to instruction above,  so think this cd should be removed

Reply:
+1